supplementary_style_guide/glossary_terms_conventions/general_conventions/m.adoc

```diff
@@ -638,7 +638,7 @@ Use "Memory Overcommitment Manager (MOM)" for the first instance in a section, a
 [discrete]
 [[multisite]]
 ==== image:images/yes.png[yes] multisite (adjective)
-*Description*: In Red Hat Ceph Storage, you can configure the Ceph Object Gateway to participate in a multisite architecture that consists of one zone group and multiple zones each zone with one or more `ceph-radosgw` instances. See the https://access.redhat.com/documentation/en/red-hat-ceph-storage/2/paged/object-gateway-guide-for-ubuntu/chapter-8-multi-site[Multisite] chapter in the Red Hat Ceph Storage 2 Object Gateway Guide for details.
+*Description*: In Red Hat Ceph Storage, you can configure the Ceph Object Gateway to participate in a _multisite_ architecture that consists of one zone group and multiple zones each zone with one or more `ceph-radosgw` instances. See the https://access.redhat.com/documentation/en/red-hat-ceph-storage/2/paged/object-gateway-guide-for-ubuntu/chapter-8-multi-site[Multisite] chapter in the Red Hat Ceph Storage 2 Object Gateway Guide for details.
```

Comment:
Link throwing a 404

Reply:
I feel this is out of scope of my changes. I added a comment in the style group. I suggested to remove it.